ignite/templates/app/app.go

```diff
@@ -24,16 +30,36 @@ func NewGenerator(opts *Options) (*genny.Generator, error) {
 		return nil, fmt.Errorf("generator sub: %w", err)
 	}
 	g := genny.New()
-	if err := g.OnlyFS(subfs, opts.IncludePrefixes, nil); err != nil {
+
+	// always exclude minimal app config it will be created later
+	// app_config_minimal is only used for the minimal app template
+	excludePrefix := []string{minimalAppConfig}
+	if opts.IsChainMinimal {
+		// minimal chain does not have ibc or classic app config
+		excludePrefix = append(excludePrefix, ibcConfig, appConfig)
+	}
+
+	if err := g.SelectiveFS(subfs, opts.IncludePrefixes, nil, excludePrefix, nil); err != nil {
 		return g, fmt.Errorf("generator fs: %w", err)
 	}
+
+	if opts.IsChainMinimal {
+		file, err := subfs.Open(fmt.Sprintf("%s.plush", minimalAppConfig))
+		if err != nil {
+			return g, fmt.Errorf("open minimal app config: %w", err)
+		}
+
+		g.File(genny.NewFile(appConfig, file))
+	}
```

Comment:
```suggestion
	excludePrefix := []string{minimalAppConfig}
	if opts.IsChainMinimal {
		// minimal chain does not have ibc or classic app config
		excludePrefix = append(excludePrefix, ibcConfig, appConfig)
		
		file, err := subfs.Open(fmt.Sprintf("%s.plush", minimalAppConfig))
		if err != nil {
			return g, fmt.Errorf("open minimal app config: %w", err)
		}

		g.File(genny.NewFile(appConfig, file))
	}

	if err := g.SelectiveFS(subfs, opts.IncludePrefixes, nil, excludePrefix, nil); err != nil {
		return g, fmt.Errorf("generator fs: %w", err)
	}
```

Reply:
Doing like this will skip the app_config, as it is excluded.